constitution.tex

```diff
@@ -742,7 +742,6 @@
 \asubsection{Qualifications to be the Chairperson, Evaluations Director}
 \begin{enumerate}
 	\item Candidates must be Active Members during the term of office
-	\item Candidates must reside on floor during the term of office
```

Comment:
If you're removing this, then merge this with the `Qualifications to be the Social Director(s), Financial Director, Research and Development Director(s), House Improvements Director, House History Director`